### PR TITLE
Add bin/www to setup Express server

### PR DESCRIPTION
**Pull request by jaysonhunter**
This file is used to launch the Express server and sets up the listener on http://localhost:5000 for development

### CODE REVIEW

File: bin/www [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const app = require('../app')
+
+// configure the app to listen for incoming
+// traffic on HOST and PORT
+const HOST = process.env.HOST || 'localhost'
+const PORT = process.env.PORT || 5000
+
+
+//-------------------------
+// set up server listening
```
Comment (LBHLMatthewsMurray) [inline bin/www:14]: Express server listening?

Comment (LBHAcochrane) [inline bin/www:14]: Testing

Comment (jaysonhunter) [reply bin/www:14]: Yes it is.

File: bin/www [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// this file is used to launch the
+// Express server and sets up the listener on
+// http://localhost:5000 for development
+
+const app = require('../app')
```
Comment (LBHLMatthewsMurray) [inline bin/www:5]: Where is this app file?

**Review by LBHLMatthewsMurray — CHANGES_REQUESTED** [review]

**Review by LBHLMatthewsMurray — APPROVED** [review]

**Review by LBHAcochrane — APPROVED** [review]

**Pull request merged**